contributors/devel/README.md

```diff
@@ -8,11 +8,8 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## The process of developing and contributing code to the Kubernetes project
 
-* **Welcome to Kubernetes (New Developer Guide)**
-  ([welcome-to-kubernetes-new-developer-guide.md](welcome-to-kubernetes-new-developer-guide.md)):
-  An introductory guide to contributing to K8s.
-
-* **On Collaborative Development** ([collab.md](collab.md)): Info on pull requests and code reviews.
+* **Contributor Guide**
+  ([Please start here](/guide/README)) to learn about how to contribute to Kuberenetes
```

Comment:
I think this should be `/contributors/guide`

Reply:
Gah. Fixed.